Tests/SwiftLayoutTests/LayoutDSLTest.swift

```diff
@@ -78,15 +78,15 @@ extension LayoutDSLTest {
     }
     
     func testSimpleWithViewConfiguration() {
-        deactivable = root {
+        root {
             UILabel().config { view in
                 view.text = "RED"
                 view.accessibilityIdentifier = "red"
                 return view
```

Comment:
I've been trying a few things, and it's practically impossible. What do you think?